server/lib/convertTime.js

```diff
@@ -0,0 +1,13 @@
+const convertTime = (number) => {
+  if (number === 0) return "12:00 AM";
+  if (number === 12) return "12:00 PM";
+  let time = `${number % 12}:00`;
+
+  if (number < 12) {
+    return (time += " AM");
+  } else {
+    return (time += " PM");
+  }
+};
+
+module.exports = { convertTime };
```

Comment:
this is just used to generate a more user friendly time format for the users?

Reply:
Yea we currently have in our table just an integer, so that is used to convert the 24 based integer into AM and PM that we are used to. I believe Christian or Steven created this function.